apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog/GoogleStackdriverTarget_Configuration.cs

```diff
@@ -98,6 +98,22 @@ public partial class GoogleStackdriverTarget
         /// </summary>
         public Layout LogId { get; set; } = "Default";
 
+        /// <summary>
+        /// Request TraceId for correlation of LogEvents from same request
+        /// </summary>
+        /// <remarks>
+        /// Resource name of the trace associated with the log entry. Ex: '06796866738c859f2f19b7cfb3214824'
+        /// </remarks>
+        public Layout TraceId { get; set; }
+
+        /// <summary>
+        /// Activity SpanId for correlation of LogEvents from same activity.
+        /// </summary>
+        /// <remarks>
+        /// Trace API v2 uses: a 16-character hexadecimal encoding of an 8-byte array. Ex: `000000000000004a` = String.format("{0:x16}", 74)
+        /// </remarks>
+        public Layout SpanId { get; set; }
+
```

Comment:
I think we'd want to add TraceSampled as well here, that should be true or false whether the trace has been sampled or not. And then we could add that value on the entry as well.

Reply:
From my side, it can be done in a separate task, sampling is something additional, however, it'd be nice. Despite that, everything looks good to me.